src/neptune/types/series/float_series.py

```diff
@@ -56,16 +61,24 @@ def __init__(
         self._unit = unit
 
         if steps is None:
-            self._steps = cycle([None])
+            filled_steps = cycle([None])
         else:
             assert len(values) == len(steps)
-            self._steps = steps
+            filled_steps = steps
 
         if timestamps is None:
-            self._timestamps = cycle([time.time()])
+            filled_timestamps = cycle([time.time()])
         else:
             assert len(values) == len(timestamps)
-            self._timestamps = timestamps
+            filled_timestamps = timestamps
+
+        clean_values, self._steps, self._timestamps = self.filter_unsupported_values(
```

Comment:
For not it's just to think of: Do you think we can refactor it someone to not to create an empty FloatSeries whenever someone will but just a single unsupported value?
I mean the following code produces an empty FloatSeries:
```python
>>> run['c'].append(float('nan'))
<stdin>:1: NeptuneUnsupportedValue: WARNING: A value you're trying to log (`nan`) will be skipped because it's a non-standard float value that is not currently supported.
>>> run.get_structure()['c']
<neptune.attributes.series.float_series.FloatSeries object at 0x13ff83d90>
```

Reply:
I think it requires a bit more rework, I would leave it for a separate pr, after trying to make that change and not looking so obvious 🤔

Reply:
Ok. Posted to Jira then